src/app/src/components/movie-details/movie-details.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="movieCard" *ngIf="movieDetails$ | async as movieDetails">
+  <div class="movieDetailsTitle">
+    <h2>
+      <b>{{ movieDetails.title }}</b>
+    </h2>
+    <button class="btn btn-primary" (click)="redirectToHomepage()">
+      Back to homepage
+    </button>
+  </div>
+  <div class="movieCardDetails">
+    <div class="movieImage">
+      <img
+        src="https://www.themoviedb.org/t/p/w220_and_h330_face{{
+          movieDetails.poster_path
+        }}"
+      />
+    </div>
+    <div class="movieDetails">
+      <h3>
+        <b>Release date:</b> {{ movieDetails.release_date | date: 'MMMM d, y' }}
+      </h3>
+      <h3>
+        <b>Original language:</b>
+        {{ movieDetails.original_language | uppercase }}
+      </h3>
+      <h3>
+        <b>Produced by:</b>
+        <span
+          *ngFor="
+            let company of movieDetails.production_companies;
+            let i = index
+          "
+        >
+          {{ company.name }}
+          <span *ngIf="i !== movieDetails.production_companies.length - 1">
```

Comment:
we dont keep logic in template so either put this in a method in component (you will have to write tests for it later)
or just do `*ngFor="let company of movieDetails.production_companies; last as last"` and `*ngIf="!last"`